src/ui/views/chrome.jade

```diff
@@ -100,7 +100,8 @@ html(lang='en')
+        /* SVG optimized according to http://codepen.io/tigt/post/optimizing-svgs-in-data-uris */
+        background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 154.03 200.11'%3E%3Ctitle%3ELogo-KibanaIcon%3C/title%3E%3Cpath opacity='0.7' fill='%23fff' d='M479.29,273.54L557,179.94H403v75c1,0,1.92-.07,2.89-0.07A152.78,152.78,0,0,1,479.29,273.54Z' transform='translate(-402.99 -179.94)'/%3E%3Cpolygon opacity='0.7' fill='%23fff' points='0 185.5 0 185.5 76.31 93.6 0 185.5'/%3E%3Cpath opacity='0.7' fill='%23fff' d='M499.17,286.52l2.14,1.66C500.61,287.62,499.89,287.07,499.17,286.52Z' transform='translate(-402.99 -179.94)'/%3E%3Cpath opacity='0.7' fill='%23fff' d='M493.27,282.19l1.22,0.87Z' transform='translate(-402.99 -179.94)'/%3E%3Cpath opacity='0.7' fill='%23fff' d='M504.4,290.74l-74,89.32H556.81A153.4,153.4,0,0,0,504.4,290.74Z' transform='translate(-402.99 -179.94)'/%3E%3Cpath opacity='0.5' fill='%23fff' d='M492.54,281.67c-5.34-3.8-13.25-8.13-13.25-8.13a153.77,153.77,0,0,1,14,8.64Z' transform='translate(-402.99 -179.94)'/%3E%3Cpath opacity='0.5' fill='%23fff' d='M504.43,290.7s-1.16-1-3.11-2.51c1,0.83,2.06,1.69,3.08,2.55Z' transform='translate(-402.99 -179.94)'/%3E%3Cpath opacity='0.5' fill='%23fff' d='M494.49,283.06c1.59,1.12,3.15,2.29,4.69,3.47C497.81,285.49,496.23,284.31,494.49,283.06Z' transform='translate(-402.99 -179.94)'/%3E%3Cpath opacity='0.5' fill='%23fff' d='M501.32,288.19l-2.14-1.66c-1.54-1.18-3.1-2.34-4.69-3.47l-1.22-.87a153.77,153.77,0,0,0-14-8.64L403,365.44v14.62h27.39l74-89.32C503.38,289.88,502.36,289,501.32,288.19Z' transform='translate(-402.99 -179.94)'/%3E%3Cpath fill='%23fff' d='M405.87,254.85c-1,0-1.92.06-2.89,0.07V365.44l76.31-91.9A152.78,152.78,0,0,0,405.87,254.85Z' transform='translate(-402.99 -179.94)'/%3E%3C/svg%3E");
```

Comment:
wow .... never knew this was even possible :goat:  .... but what are the reasons for having the svg inside the css ? (instead of being a separate document or at least having it inside html ? )